packages/serve/src/index.ts

```diff
@@ -12,6 +12,9 @@ import parseArgs from './parseArgs';
 export default function serve(...args: string[]): void {
     const cli = new WebpackCLI();
 
+    // add WEBPACK_SERVE to compilation environment
+    args = [...args, '--env', 'WEBPACK_SERVE=true'];
```

Comment:
Maybe we can avoid it? It can be unsafe...

Reply:
Updated, see if fine now.

P.S.
>  It can be unsafe

Can you explain this a bit or point me to a place where can I learn more about this?

Reply:
@evilebottnawi can you give example?

Reply:
we remove this code :smile: